### DISCUSSION TITLE
How to adjust fixed labels in ODF?

### DISCUSSION THREAD

**Discussion by trudator** [mtex-toolbox/mtex#1648] [Ask Anything] [answered]
Hi folks, 

The minmax and phi2 values are sort of covered inside my ODF plots. How could I change these fixed labels to the outside of the plot (let's say move the max and phi2 values to the top and the min value to the bottom)?

Thanks

![image](https://user-images.githubusercontent.com/117386810/201491611-7409f474-319a-471a-b34d-e25578ec688d.png)

**Comment by kilir** [comment, marked as answer]
Hi,
how about:
```
mtexdata dubna
odf = pf.calcODF

plotSection(odf,'minmax')
f= gcm;
for i= 1:length(f.children)
sP = getappdata(f.children(i),'sphericalPlot')
sP.TL.Position = [0.01 1.25 0];
sP.BL.Position = [0.1 1.0 0];
sP.TR.Position = [0.9950 1.15 0];
end
```
Adapt the values to your needs.
Cheers,
Rüdiger

**Reply by trudator** [reply]
Thanks Rudiger!

**Reply by SunnySharma200** [reply]
Hello @trudator ,

Can you please tell me how the code sent by @kilir has helped you? I have used the same in my code but it didn't help. 
____________________________________________________________
**%% Plotting ODF's Phi1 sections**
ori = calcDensity(ebsd.orientations);
figure(5)
plotSection(ori,'contourf','FontSize',30,'FontWeight','bold','TextBackground','black','linewidth',0.5,...
    'TextColor','white','phi1',[0,30,60,90]*degree,'silent'...
    ,'linecolor','black','linewidth',2,'ShowText','off','minmax');
for i= 1:length(f.children)
    sP = getappdata(f.children(i),'sphericalPlot');
    sP.TL.Position = [0.1 1.25 0];
    sP.BL.Position = [0.1 1.0 0];
    sP.TR.Position = [0.9950 1.15 0];
end
set(findall(gcf,'type','axes'),'FontSize',25,'Fontname','Times New Roman');
set(findall(gcf,'type','text'),'FontSize',25,'Fontname','Times New Roman','FontWeight','bold');
% mtexColorMap white2black ;
h1 = mtexColorbar;
set(h1,'FontSize',25,'Fontname','Times New Roman');
CLim(gcm,[0,16]); % set equal color range
